docs/QUICKSTART.md

```diff
@@ -0,0 +1,418 @@
+---
+layout: default
+title: "Quickstart"
+nav_order: -4
+---
+
+# Quickstart
+
+## Requirements
+
+- [Node.js v16.15.0 or latest stable](https://nodejs.org/en/). We recommend using [nvm](https://github.com/nvm-sh/nvm) to install.
+
+## Introduction
+
+In this tutorial we are going to;
+
+1. 🌴 Install `forc` and the `fuel-core` locally
+2. ⚡ Running a Fuel locally
+3. 💻 Write & Deploy a **🌴 Sway** Contract
+4. 💚 Create a Fuel DApp to interact with your Contract
+
+You can also access the [Example App here](./examples/my-fuel-dapp).
+
+## 🌴 Install `forc` and the `fuel-core` locally
+
+`Forc` is similar to "npm", or "cargo" but for [**🌴 Sway**](https://fuellabs.github.io/sway). `Forc` stands for Fuel Orchestrator. Forc provides a variety of tools and commands for developers working with the Fuel ecosystem, such as scaffolding a new project, formatting, running scripts, deploying contracts, testing contracts, and more. If you're coming from a Rust background, `forc` is similar to cargo.
+[read more about Forc](https://fuellabs.github.io/sway/v0.19.1/forc/index.html)
+
+`Fuel Core` is the implementation of the `Fuel VM`. `Fuel Core` provides the ability to spin-up a `Fuel Client` locally with custom chain configs. The `Fuel Core` is also used on the live chains, but we are not going to cover it here, as we wan't to focus on development only.
+[See more](https://github.com/FuelLabs/fuel-core)
+
+### MacOS and Linux
+
+When using `MacOS` and `Linux`. We provide a tools [`fuelup`](https://github.com/FuelLabs/fuelup) that works as toolchain manager for Fuel.
+
+#### 1. Install `fuelup`;
+
+```sh
+curl --proto '=https' --tlsv1.2 -sSf https://fuellabs.github.io/fuelup/fuelup-init.sh | sh
```

Comment:
Is there a way to add a copy button to this terminal command and the other terminal commands in general?  That's always a nice feature for devs working through docs.

Reply:
I add a button now